<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCommentWindow</class>
 <widget class="QDialog" name="AddCommentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>撰写评价</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(250,250,250);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,7">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,3,1,1">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>为课程撰写评价</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sl_rating">
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TickPosition::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_rating">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>评分：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="BetterButton" name="btn_submit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>提交</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="te_comment">
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255,255,255);</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BetterButton</class>
   <extends>QPushButton</extends>
   <header>betterbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
